odoo/tools/config.py

```diff
@@ -164,22 +164,22 @@ def __init__(self, fname=None):
                          help="Enable unit tests.")
         group.add_option("--test-tags", dest="test_tags",
                          help="""Comma-separated list of specs to filter which tests to execute. Enable unit tests if set.
-                         A filter spec has the format: [-][tag][/module][:class][.method]
-                         The '-' specifies if we want to include or exclude tests matching this spec.
-                         The tag will match tags added on a class with a @tagged decorator
-                         (all Test classes have 'standard' and 'at_install' tags
-                         until explicitly removed, see the decorator documentation).
-                         '*' will match all tags.
-                         If tag is omitted on include mode, its value is 'standard'.
-                         If tag is omitted on exclude mode, its value is '*'.
-                         The module, class, and method will respectively match the module name, test class name and test method name.
-                         examples: :TestClass.test_func,/test_module,external
-
-                         Filtering and executing the tests happens twice: right
-                         after each module installation/update and at the end
-                         of the modules loading. At each stage tests are filtered
-                         by --test-tags specs and additionally by dynamic specs
-                         'at_install' and 'post_install' correspondingly.""")
+A filter spec has the format: [-][tag][/module][:class][.method]
+The '-' specifies if we want to include or exclude tests matching this spec.
+The tag will match tags added on a class with a @tagged decorator
+(all Test classes have 'standard' and 'at_install' tags
+until explicitly removed, see the decorator documentation).
+'*' will match all tags.
+If tag is omitted on include mode, its value is 'standard'.
+If tag is omitted on exclude mode, its value is '*'.
+The module, class, and method will respectively match the module name, test class name and test method name.
+Example: --test-tags :TestClass.test_func,/test_module,external
+
+Filtering and executing the tests happens twice: right
+after each module installation/update and at the end
+of the modules loading. At each stage tests are filtered
+by --test-tags specs and additionally by dynamic specs
+'at_install' and 'post_install' correspondingly.""")
```

Comment:
I prefer single-double quotes string to keep the indentation :thinking:.
```python
                         help="..."
                         "..."
                         "..."
                         "...")
```

Reply:
done 
![image](https://user-images.githubusercontent.com/186131/119832930-f2d5ca80-befe-11eb-9cbc-d606cd4a028d.png)